src/main/resources/modules/metabolic_syndrome/kidney_conditions.json

```diff
@@ -609,8 +609,8 @@
       "codes": [
         {
           "system": "LOINC",
-          "code": "24321-2",
-          "display": "Basic Metabolic 2000 Panel"
+          "code": "51990-0",
+          "display": "Basic metabolic panel - Blood"
```

Comment:
I think this is supposed to be `24321-2`, because it allows for the optional inclusion of `Glomerular filtration rate`.

The `51990-0` panel does not.

Reply:
Ahh I didn't notice that difference. The issue here is if you look at the codes for the individual observations they are all "X in Blood" vs "X in Serum/Plasma". For example right below this here is `"2339-0" Glucose [Mass/volume] in Blood` but then in the gallstones module you'll see `"2345-7" Glucose [Mass/volume] in Serum or Plasma`. So I can change this back to the 24321-2 panel but then I think the individual component codes all have to be changed to the "serum" versions as well.

Reply:
As long as the UOM are correct, then we should probably do that.

Reply:
Ok - updated all 5 panels in this file to be `24321-2` and changed the codes to be the corresponding "in serum" observations. All the units matched though the one for that Glomerular filtration rate did appear to have the wrong unit. We use that code in a bunch of other modules and they all had unit = `mL/min/{1.73_m2}` so I switched it to that